src/aelfrice/transcript_logger.py

```
@@ -202,6 +204,18 @@ def _handle_user_prompt_submit(payload: dict[str, object]) -> None:
+    # #747: harness-wrapper prompts (<task-notification>, <summary>Monitor,
+    # <tool-result>, etc.) carry no user intent and crowd real turns out of
+    # the rebuilder's recent-turns window. Gate the append on the same
+    # noise predicate ingest.py uses; fail-soft so a noise_filter import
+    # error never breaks the logger.
+    try:
+        from aelfrice.noise_filter import is_transcript_noise  # noqa: PLC0415
+
+        if is_transcript_noise(prompt):
+            return
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)